### ISSUE TITLE
Unexpected Exception: unhashable type: 'AnsibleSequence'

### ISSUE THREAD

**Issue by vboginskey** [ansible/ansible#14157]
##### Issue Type:

Bug Report
##### Ansible Version:

```
ansible 2.0.0.2
  config file =
  configured module search path = Default w/o overrides
```
##### Ansible Configuration:

Default
##### Environment:

OS X 10.10.5
##### Summary:

Providing a list of lists as a role param dumps a traceback.
##### Steps To Reproduce:

```
❯ tree
.
├── roles
│   └── debug
│       └── tasks
│           └── main.yml
└── test.yml

❯ cat roles/debug/tasks/main.yml

---
- debug: var=params

❯ cat test.yml

---
- name: Reproduce Ansible bug

  hosts: localhost
  connection: local
  gather_facts: false

  roles:
    - role: debug
      params:
        - [ 'foo', 'bar' ]
```
##### Expected Results:

```
ok: [localhost] => {
    "params": [
        [
            "foo",
            "bar"
        ]
    ]
}
```
##### Actual Results:

```
No config file found; using defaults
 [WARNING]: provided hosts list is empty, only localhost is available

Unexpected Exception: unhashable type: 'AnsibleSequence'
the full traceback was:

Traceback (most recent call last):
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/bin/ansible-playbook", line 85, in <module>
    sys.exit(cli.run())
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/cli/playbook.py", line 150, in run
    results = pbex.run()
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/executor/playbook_executor.py", line 79, in run
    pb = Playbook.load(playbook_path, variable_manager=self._variable_manager, loader=self._loader)
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/playbook/__init__.py", line 52, in load
    pb._load_playbook_data(file_name=file_name, variable_manager=variable_manager)
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/playbook/__init__.py", line 92, in _load_playbook_data
    entry_obj = Play.load(entry, variable_manager=variable_manager, loader=self._loader)
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/playbook/play.py", line 109, in load
    return p.load_data(data, variable_manager=variable_manager, loader=loader)
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/playbook/base.py", line 200, in load_data
    self._attributes[name] = method(name, ds[name])
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/playbook/play.py", line 211, in _load_roles
    roles.append(Role.load(ri, play=self))
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/playbook/role/__init__.py", line 104, in load
    hashed_params = hash_params(params)
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/playbook/role/__init__.py", line 56, in hash_params
    s.update((k, tuple(things)))
TypeError: unhashable type: 'AnsibleSequence'
```

**Comment by jimi-c** [comment]
Hi @vboginskey this has been corrected in the devel branch, and will be included in 2.0.1 (we're working on getting RC1 out for that very soon).

**Comment by vboginskey** [comment]
Great, thank you for the quick follow-up!
